framework/encode/custom_vulkan_api_call_encoders.cpp

```diff
@@ -542,6 +550,161 @@ VKAPI_ATTR VkResult VKAPI_CALL CreateGraphicsPipelines(VkDevice
     VkResult result = vulkan_wrappers::GetDeviceTable(device)->CreateGraphicsPipelines(
         device, pipelineCache, createInfoCount, pCreateInfos_unwrapped, pAllocator, pPipelines);
 
+    // for Linking Graphics Piplelines Libraries start[--
+#if defined(VK_USE_PLATFORM_ANDROID_KHR)
+    if (pCreateInfos->flags == 0x0 && pCreateInfos->pNext != nullptr)
+    {
+        const VkPipelineCreationFeedbackCreateInfo* feedbackInfo =
+            (reinterpret_cast<const VkPipelineCreationFeedbackCreateInfo*>(pCreateInfos->pNext));
+        const VkPipelineLibraryCreateInfoKHR* libraryInfo =
+            (reinterpret_cast<const VkPipelineLibraryCreateInfoKHR*>(feedbackInfo->pNext));
+        if (libraryInfo != nullptr)
+        {
+            if (libraryInfo->libraryCount == 4)
+            {
+                GFXRECON_LOG_WARNING("[capture]vkCreateGraphicsPipelines:link:libraryCount=%d,pLibraries[0]=%p,"
```

Comment:
These look like debugging messages, and if so, probably they should be `GFXRECON_LOG_DEBUG`.  If these are generally for debugging and testing this PR, please consider making the PR a draft and consider removing the debugging statements before moving from draft to final.